src/common/bustub_instance.cpp

```diff
@@ -32,7 +32,7 @@ BustubInstance::BustubInstance(const std::string &db_file_name) {
   // Log related.
   log_manager_ = new LogManager(disk_manager_);
 
-  buffer_pool_manager_ = new BufferPoolManagerInstance(BUFFER_POOL_SIZE, disk_manager_, LRUK_REPLACER_K, log_manager_);
+  buffer_pool_manager_ = new BufferPoolManagerInstance(128, disk_manager_, LRUK_REPLACER_K, log_manager_);
```

Comment:
I remember you mentioned needing to change this; please document hardcoded constants (in a nearby comment above is fine)